content/v2.0/reference/flux/stdlib/experimental/geo/filterrows.md

```diff
@@ -35,6 +35,27 @@ geo.filterRows(
 )
 ```
 
+### Strict and non-strict filtering
+In most cases, the specified geographic region does not perfectly align with S2 grid cells.
+**Non-strict filtering** returns points that may be outside of the specified region but
```

Comment:
maybe start w Strict filtering sentence?

- **Strict filtering** returns points inside the specified (filter?) region.
- **Non-strict filtering** returns points inside S2 grid cells covered by the specified region.

....not sure about adding "filter" in text, maybe extra? Either way, match text to graphic